githubteacher-orlando.md

```diff
@@ -0,0 +1,10 @@
+# Orlando, FL
+
+### Restaurants
+
+- The Pharmacy
+- 4Rivers
+
+### Tourist Attractions
```

Comment:
@githubteacher Orlando is big for Theme Parks -- isn't it? Shouldn't you include that?